docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -0,0 +1,79 @@
+---
+description: To prevent "resource not accessible by integration" error when running job to upload findings to GitHub's Advanced Security Dashboard
+tags:
+  - Semgrep CI
+  - GitHub
+---
+
+## Why aren't findings populating in the GitHub Advanced Security Dashboard after running Semgrep in CI?
+
+When scanning with Semgrep in CI, findings automatically populate in the Semgrep Cloud Platform. To show findings in the GitHub Advanced Security Dashboard, you must first run an alternate job that uploads findings in the form of a `SARIF` file to the dashboard. See  [Sample GitHub Actions configuration file](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file) for an example.
+
+Once you run the alternate job, the job may fail and return a "resource not accessible by integration" error which has two possible causes:
+
+**1. Your repo is private**
```

Comment:
```suggestion
## Your repository is private
```
I also prefer repo, but our style guide generally prefers full usages at least once first - so you can do `repository (repo)` or just go with repository.

For the list > heading, this will make the typography more prominent and avoid suggesting that the issues are sequenced when they're actually independent.

Reply:
<img width="738" alt="Snag_4807f1f" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/8065e910-d9db-4432-aa53-6fde052d93e3">

The reason for that convention has partly to do with writing for a global audience. Not all of our users speak English natively, so spelling out the whole word is easier for them if they use a translation service such as Google Translate or are unfamiliar with informal English.

Here's the wordlist source: [Google word list](https://developers.google.com/style/word-list)